Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Apsitvarkom.Models.DTO;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Apsitvarkom.Api.Controllers;
+
+[ApiController]
+[Route("/api/[controller]")]
+public class PollutedLocationController : ControllerBase
+{
+    private readonly ILogger<PollutedLocationController> _logger;
+
+    public PollutedLocationController(ILogger<PollutedLocationController> logger)
+    {
+        _logger = logger;
+    }
+
+    [HttpGet]
+    [Route("/[controller]/All")]
+    public ActionResult<IEnumerable<PollutedLocationDTO>> GetAll()
+    {
+        IEnumerable<PollutedLocationDTO> testData = new List<PollutedLocationDTO>()
+        {
+            new PollutedLocationDTO
+            {
+                Id = "5be2354e-2500-4289-bbe2-66210592e17f",
+                Location = new LocationDTO
+                {
+                    Latitude = 12.1651,
+                    Longitude = 121.151,
+                },
+                Radius = 10,
+                Severity = "MODERATE",
+                Spotted  =  DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.ttt"),
```

Comment:
This timestamp format is not ISO8601 format, use `DateTime.UtcNow.ToString("o", CultureInfo.InvariantCulture)` to get this format: `"2022-09-22T09:00:29.4542617Z"`, like we specified in #4

Reply:
Of course use this for both DTOs you're creating

Reply:
Refer to [this SO question](https://stackoverflow.com/questions/114983/given-a-datetime-object-how-do-i-get-an-iso-8601-date-in-string-format)

Reply:
To have more diversified data in DTOs I would have suggested we could have hard-coded one of the strings and not used two identical datetimes, although that's also good.